### DISCUSSION TITLE
Where can I find character generation prompts?

### DISCUSSION THREAD

**Discussion by fdtory** [oobabooga/text-generation-webui#2170] [Q&A]
Hello,

I am well aware of the character sheet for role playing on Oobabooga, but I would like to know how Oobabooga calls for description, first message, personality, description... all these bits.

Is there anywhere I can find this part of the code in this repository?

Thanks a lot.

**Comment by m-spangenberg** [comment]
Check here: https://github.com/oobabooga/text-generation-webui/blob/f052ab9c8fed3dedc446c3847f10ab22e42bfb37/modules/chat.py#L472
